internal/instruments.go

```diff
@@ -0,0 +1,44 @@
+package internal
+
+import (
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/api/unit"
+	"log"
+)
+
+type openTelemetryInstrumentation struct {
+	// Count of write commands performed
+	WriteCount metric.Int64Counter
+	// Count of new connections
+	NewConnectionsCount metric.Int64Counter
+}
+
+var Instruments = initInstruments()
+
+func initInstruments() *openTelemetryInstrumentation {
+	meter := global.Meter("github.com/go-redis/redis")
+
+	writeCount, err := meter.NewInt64Counter("redis.num_writes",
+		metric.WithDescription("the number of writes initiated"),
+		metric.WithUnit(unit.Bytes),
+	)
+	if err != nil {
+		// TODO: handle errors
+		log.Printf("failed to create instrument WriteCount")
+	}
```

Comment:
Probably a better option to use the project logger here: https://github.com/go-redis/redis/blob/master/internal/log.go#L12

E.g.

```suggestion
	if err != nil {
		// TODO: handle errors
		Logger.Printf("failed to create instrument WriteCount")
	}
```